lib/src/sql/with.rs

```diff
@@ -2,14 +2,15 @@ use crate::sql::comment::shouldbespace;
 use crate::sql::common::commas;
 use crate::sql::error::IResult;
 use crate::sql::ident::ident_raw;
-use derive::Store;
 use nom::branch::alt;
 use nom::bytes::complete::tag_no_case;
 use nom::multi::separated_list1;
+use revision::revisioned;
 use serde::{Deserialize, Serialize};
 use std::fmt::{Display, Formatter, Result};
 
-#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Store, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash)]
```

Comment:
Everything else only had the macro call added, but this one also had `Store` derive removed, why is that?

Reply:
The `sql::With` type never gets serialised to disk directly, so the `Store` derive wasn't needed on this type in the first place!